scripts/export.md

```diff
@@ -0,0 +1,181 @@
+and bitstreams (if applicable).
+
+## Parameters
+
+It takes several parameters to customize the export, such as the type of object to export, the ID or handle of the
+object, the sequence number to begin exporting items with, whether to export for migration, and whether to exclude
+bitstreams.
+
+The following parameters are available for the export script:
+
+| Parameter                         | Description                                                                              |
+|-----------------------------------|------------------------------------------------------------------------------------------|
+| `-t`, <br/>`--type <value>`       | The type of item to export. Valid values include: COLLECTION or ITEM.                    |
+| `-i`, <br/>`--id <value>`         | The ID or handle of the thing to export.                                                 |
+| `-n`, <br/>`--number <value>`     | The sequence number to begin exporting items with.                                       |
+| `-m`, <br/>`--migrate`            | Export for migration (remove handle and metadata that will be re-created in new system). |
+| `-x`, <br/>`--exclude-bitstreams` | Do not export bitstreams.                                                                |
+| `-h`, <br/>`--help`               | Show help.                                                                               |
+
+The type parameter specifies whether to export a collection or an item. If exporting a collection, the script will
+export all items within the collection. The ID parameter specifies the ID or handle of the item or collection to be
+exported.
+
+The number parameter is used to specify the sequence number to begin exporting items with. This parameter is useful when
+exporting large collections, as it allows you to export items in batches rather than exporting the entire collection at
+once.
+
+The migrate parameter is used to export items for migration. When exporting items for migration, the script will remove
+the handle and metadata that will be re-created in the new system.
+
+The exclude-bitstreams parameter is used to exclude bitstreams from the export. This parameter can be useful when
+exporting large collections that contain many large files, as it can reduce the size of the resulting SAF file.
+
+## Conclusion/QuickGuide:
+
+The export script is a useful tool for batch exporting items from DSpace to Simple Archive Format (SAF). By default, the
+script exports both metadata and bitstreams, but can be customized to exclude bitstreams and metadata as needed. If you
+need to migrate your DSpace repository to a new system, the export script can be used to export items in a format that
+can be easily imported into the new system.
+
+# What is SAF?
+
+## DSpace Simple Archive Format
+
+The basic concept behind the DSpace's Simple Archive Format is to create an archive, which is a directory containing one
```

Comment:
Small thing, this should link back the main documentation for SAF at https://wiki.lyrasis.org/display/DSDOC7x/Importing+and+Exporting+Items+via+Simple+Archive+Format

Reply:
Is this simply additional information added to the end of the paragraph, or is it intended to replace the provided text? Thanks for the feedback!

Reply:
@Sondissimo : 

You can always just add a sentence on the end that says something like:
"For more information on the Simple Archive Format see the DSpace Documentation."  (and add the link in that sentence).

The same goes with the Discovery link (mentioned below).  Basically, I think we should add some sort of reference back to the DSpace documentation when there's a related page (similar to what you've added in some of the other *.md files in this same PR)